transform.go

```diff
@@ -188,6 +189,30 @@ func (t *Transform) GaussBlur(sigma float64) *Transform {
 	return t
 }
 
+// Rotate rotates image by a multiple of 90 degrees
+func (t *Transform) Rotate(angle Angle) *Transform {
+	t.tx.Rotate = angle
+	return t
+}
+
+// Rotate90 rotates image by 90 degress clockwise
+func (t *Transform) Rotate90() *Transform {
```

Comment:
I don't think we need these separate methods, just the `Rotate` with the given `Angle`